src/views/OfflineDatalog.vue

```diff
@@ -298,8 +356,31 @@ export default {
         cmdList[CONST.command_id_index] = CONST.rcmd_clear_offline_datalog;
 
         this.sendCommand(cmdList, null);
+        alert("Data in GoGoBoard has been deleted");
+        this.$vm2.close("modal");
       }
     },
+    //? Add function for refresh date on you pick
+    onSelectedDate() {
+      let renderdata = structuredClone(
+        this.datalogRecords[this.selectedChannel["name"]]
+      ); //Deep Copy
+      console.log(renderdata);
+      renderdata.forEach((field) => {
+        for (let i = 0; i < field["data"].length; i++) {
+          field["data"][i][0] += this.dateTimeOffset;
+        }
+        return field["data"];
+      });
+      this.$refs.datalogChart.chartOptions.series = renderdata;
+    },
+    doDelete() {
+      this.$vm2.close("modal-1");
+      this.$vm2.close("modal-3");
+    },
+    close() {
+      this.$vm2.close("modal-4");
+    },
```

Comment:
not sure what these modals name mean ? can you describe how this being used ?

Reply:
Sorry, It's both test functions. It can remove both